hack/cli-testing-environment.sh

```diff
@@ -38,6 +38,7 @@ hack/create-cluster.sh ${MEMBER_CLUSTER_2_NAME} ${KUBECONFIG_PATH}/${MEMBER_CLUS
 # wait cluster ready
 echo "Wait clusters ready..."
 util::wait_file_exist ${KUBECONFIG_PATH}/${HOST_CLUSTER_NAME}.config 300
+util::wait_context_exist ${HOST_CLUSTER_NAME} ${KUBECONFIG_PATH}/${HOST_CLUSTER_NAME}.config 300
```

Comment:
Not only host clusters, there is other two member clusters.

Reply:
Done.